### PR TITLE
Add permission group mapping

### PR DESCRIPTION
**Pull request by soerenbe**
Proposal and discussion see #37

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -103,4 +103,10 @@ At this point, the django-shibboleth-remoteuser middleware should be complete.
     )
    ```
 
+### Permission group mapping
+ * It is possible to map a list of attributes to Django permission groups. ```django-shibboleth-remoteuser``` will generate the groups from the semicolon-separated values of this attributes. They will be available in the Django admin interface and you can assign your application permission to them.
```
Comment (bcail) [inline README.md]: "this attributes" => "these attributes". "application permission" => "application permissions"

File: README.md [hunk 2]
```diff
@@ -103,4 +103,10 @@ At this point, the django-shibboleth-remoteuser middleware should be complete.
...
 
+   ```python
+   SHIBBOLETH_GROUP_ATTRIBUTES = ['Shibboleth-affiliation', 'Shibboleth-isMemberOf']
+   ```
+ By default this value is empty and will not affect your group settings. But when you add attributes to ```SHIBBOLETH_GROUP_ATTRIBUTES``` the user will only associated with those groups. Be aware that the user will removed from groups not defined in ```SHIBBOLETH_GROUP_ATTRIBUTES```, if you enable this setting. Some installations may create a lot of groups. You may check your group attributes at [https://your_domain.edu/Shibboleth.sso/Session]() before activating this feature.
```
Comment (jrhoads) [inline README.md]: "the user will removed" =>"the user will be removed"

**Comment by jrhoads** [comment]
Great feature.  Thank you for your hard work.

**Comment by soerenbe** [comment]
I have updated the PR and fixed the typos.

Thanks for your feedback.

**Pull request merged**